ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -290,7 +290,202 @@ <h3 class="title">
         </table>
       </div>
     </mat-tab>
-    <mat-tab label="PRIMARY KEY"></mat-tab>
+    <mat-tab label="PRIMARY KEY">
+      <div class="pk-tab-container">
```

Comment:
possible to remove empty cells after primary key deletion : https://drive.google.com/file/d/1zuwEiV-yscZa2l1jHvRDduTEXx6ELlx_/view?usp=sharing&resourcekey=0-3RJsYlnSX1dW4QYUd_7IbA . User might get confused with these

Reply:
Please provide a little more clarity on this issue? When are we getting this exactly?

Reply:
I understood the case , you can ignore this comment